doc/articles/controls/SkiaCanvas.md

```diff
@@ -0,0 +1,56 @@
+---
+uid: Uno.Controls.SKCanvasElement
+---
+
+# Introduction
+
+During creating an Uno application, users might want to create elaborate 2D graphics that are more suitable to a 2D graphics library such as [Skia](https://skia.org) or [Cairo](https://www.cairographics.org), rather than using, for example, a simple [Canvas](https://learn.microsoft.com/en-us/windows/windows-app-sdk/api/winrt/microsoft.ui.xaml.controls.canvas). To support this use case, SkiaSharp comes with an [SKXamlCanvas](https://learn.microsoft.com/en-us/dotnet/api/skiasharp.views.windows.skxamlcanvas?view=skiasharp-views-2.88) element that allows for drawing in an area using SkiaSharp.
+
+On Uno Skia targets, we can utilize the pre-existing Skia canvas that is used internally by Uno to render the application instead of creating additional Skia surfaces and then copying the resulting renderings to the application (e.g. using a [BitmapImage](https://learn.microsoft.com/en-us/windows/windows-app-sdk/api/winrt/microsoft.ui.xaml.media.imaging.bitmapimage)). This way, a lot of Skia functionally can be acquired "for free". For example, no additional setup for OpenGL is needed if the Uno application is already using OpenGL to render.
```

Comment:
```suggestion
On Uno Skia targets, you can utilize the pre-existing Skia canvas that is used internally by Uno to render the application instead of creating additional Skia surfaces and then copying the resulting renderings to the application (e.g. using a [BitmapImage](https://learn.microsoft.com/windows/windows-app-sdk/api/winrt/microsoft.ui.xaml.media.imaging.bitmapimage)). This way, a lot of Skia functionally can be acquired "for free". For example, no additional setup for OpenGL is needed if the Uno application is already using OpenGL to render.
```